src/libexpr/include/nix/expr/value.hh

```diff
@@ -1119,6 +1301,7 @@ public:
         return getStorage<Lambda>();
     }
 
+    // FIXME: remove this since reading it is racy.
```

Comment:
Is this just a matter of porting the call sites to something else, or is there something more "interesting" that needs to be taken into account before this can be fixed?